react/features/filmstrip/functions.native.js

```diff
@@ -61,6 +70,22 @@ export function shouldRemoteVideosBeVisible(state: Object) {
             || disable1On1Mode);
 }
 
+/**
+ * Returns the number of participants displayed in tile view.
+ *
+ * @param {Object | Function} stateful - The Object or Function that can be
+ * resolved to a Redux state object with the toState function.
+ * @returns {number} - The number of participants displayed in tile view.
+ */
+export function getTileViewParticipantCount(stateful: Object | Function) {
+    const state = toState(stateful);
+    const disableSelfView = shouldHideSelfView(state);
+    const localParticipant = getLocalParticipant(state);
+    const participantCount = getParticipantCountWithFake(state) - (disableSelfView && localParticipant ? 1 : 0);
```

Comment:
When would there not be a local participant? Why is the `disableSelfView` check not enough?

Reply:
At some point I was receiving some errors that the local participant was `undefined` and I got the habit of adding a safety check :)

I don't remember if this was happening here but generally some functions can be executed before the local participant is created. So I find it as a good practice.

In this case when I checked it now, the function is used as a condition in a selector and therefore I think it is possible to have it executed before the local participant is created.